features/include/pcl/features/impl/esf.hpp

```diff
@@ -452,7 +452,7 @@ pcl::ESFEstimation<PointInT, PointOutT>::voxelize9 (PointCloudIn &cluster)
 template <typename PointInT, typename PointOutT> void
 pcl::ESFEstimation<PointInT, PointOutT>::cleanup9 (PointCloudIn &cluster)
 {
-  for (std::size_t i = 0; i < cluster.points.size (); ++i)
+  for (std::size_t i = 0; i < cluster.size (); ++i)
```

Comment:
for-range

Reply:
What's the code in the ternary operator doing?
```cpp
int xx = x < 0 ? floor(x) + CONST : (ceil(x) - 1) + CONST;
```
Isn't `floor(x)` same as `ceil(x) - 1`, regardless of sign of `x`?